src/scene/vcVerticalMeasureTool.cpp

```diff
@@ -64,8 +63,7 @@ void vcVerticalMeasureTool::EndMeasure(vcState *pProgramState, const udDouble3 &
 
 void vcVerticalMeasureTool::OnNodeUpdate(vcState *pProgramState)
 {
-  vdkProjectNode_GetMetadataBool(m_pNode, "pickStart", &m_pickStart, true);
-  vdkProjectNode_GetMetadataBool(m_pNode, "pickEnd", &m_pickEnd, true);
+  vdkProjectNode_GetMetadataInt(m_pNode, "selectedPoint", &m_selectedPoint, -1);
```

Comment:
@OsiJr Should we be handling the old metadata going forward?

Reply:
Yes but why are these attributes even being saved to the project?

Reply:
Sorry I thought we needed to save all attributes, will remove it.

Reply:
Removed.